### ISSUE TITLE
Selecting end time for embed feature

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH-MLA/AAPB2#1949]
As a user, I should be able to indicate an end time so that the embedded video will end at the indicated time, rather than play through the full recording.
